core/src/schema/scenario.rs

```diff
@@ -0,0 +1,167 @@
+use anyhow::{Context, Result};
+use std::path::PathBuf;
+
+use crate::Namespace;
+
+pub struct Scenario {
+    namespace: Namespace,
+    scenario: Namespace,
+    name: String,
+}
+
+impl Scenario {
+    pub fn new(namespace: Namespace, namespace_path: PathBuf, scenario: &str) -> Result<Self> {
+        let scenario_path = namespace_path
```

Comment:
I've been playing around with the binary and my natural instinct was to do something like `--scenario users/scenarios/users.json`. Do you think it's worth us supporting both. On the other hand this would be incoherent with the `--collection` interface where you don't specify a collection file, but the collection name. Food for thought.